ci/config.xml

```diff
@@ -22,10 +22,10 @@ Continuous integration of the F
 &lt;em&gt;Note:&lt;/em&gt; Contact Michael.D.Starch@jpl.nasa.gov for comments and/or questions!</description>
   <keepDependencies>false</keepDependencies>
   <properties>
-    <com.coravy.hudson.plugins.github.GithubProjectProperty plugin="github@1.29.3">
+    <com.coravy.hudson.plugins.github.GitHubProjectProperty plugin="github@1.29.3">
```

Comment:
GitHub is a brand..

That said, this is:
https://github.com/jenkinsci/github-plugin/blob/master/src/main/java/com/coravy/hudson/plugins/github/GithubProjectProperty.java

So, I'll have to drop this (as it clearly won't work) and have to at some point see about sending them a PR (which would have to include backwards compatibility). (I have contributed to that repo in the past, but apparently haven't done a spelling pass...)

Reply:
Agreed